core/lib/state/src/postgres/mod.rs

```diff
@@ -489,11 +491,26 @@ impl ReadStorage for PostgresStorage<'_> {
             values_cache.and_then(|cache| cache.get(self.l2_block_number, hashed_key));
 
         let value = cached_value.unwrap_or_else(|| {
+            const RETRY_INTERVAL: Duration = Duration::from_millis(500);
+            const MAX_TRIES: usize = 20;
+
```

Comment:
Could we mark it as to do for creating more generic solution?

Reply:
now, it's applied only for reading value and we can't configure it.

Reply:
I'm not sure that we need more generic thing here.. If in future there will be need for it then we will add, but for now I don't think it's needed